libtiny_client/src/state.rs

```diff
@@ -255,7 +265,7 @@ impl StateInner {
                 let param = &params[1];
                 match param.find('=') {
                     None => {
-                        // TODO: Log this
+                        error!("Could not parse 302 RPL_USERHOST to set usermask.");
```

Comment:
Following https://stackoverflow.com/a/2031209/691032 I think this should be `warn` because it doesn't break anything, it's just an interesting case.